src/deployment/flavors.md

```diff
@@ -153,18 +153,24 @@ project 'Runner', {
 Setting up flavors in Android can be done in your project's 
 **build.gradle** file.
 
-1. In VSCode, inside your Flutter project, 
-navigate to **android**/**app**/**build.gradle**.     
+1. Inside your Flutter project, 
+   navigate to **android**/**app**/**build.gradle**.
 
-2. Add a flavor object with the specified environments along with values for 
-**dimension**, **resValue**, and **applicationId** or **applicationSuffix**. 
-  * The name of the application for each build is located in **resValue** 
-(as shown in the snippet below from VSCode).
-  * If you specify a **applicationSuffix** instead of a **applicationId**, 
-    it is appended to the "base" application id.
+2. Create a [`flavorDimension`][] to group your added product flavors.
+   Gradle doesn't combine product flavors that share the same `dimension`.
+
+3. Add a `productFlavors` object with the desired flavors along
+   with values for **dimension**, **resValue**,
+   and **applicationId** or **applicationSuffix**.
+
+   * The name of the application for each build is located in **resValue**.
+   * If you specify a **applicationSuffix** instead of a **applicationId**, 
+     it is appended to the "base" application id.
```

Comment:
@parlough : I'm unsure where my comments on this went (maybe I didn't hit send?), but there are a few issues here.

**issues:** 

1. Everything in **bold** should be in `monospace` per the [GSG](https://developers.google.com/style/code-in-text).
2. Clarify what "desired flavors" are. This should be a separate sentence along the lines of "Within this `productFlavors` object, add one flavor with the following options."
3. Clarify what each option includes. You cover two, but omit the others. A table may be a better option here.

Reply:
I agree with these issues and plan to address them but they are pre-existing and not introduced by this PR and are present across the whole page, so would need some larger changes for consistency. 

If possible, I would like to get these fixes in right now as the details modified here were incorrect and address those issues alongside other page fixes.